ChatController.java

```diff
@@ -0,0 +1,100 @@
+package lesson2_4.online;
+
+import javafx.application.Platform;
+import javafx.collections.FXCollections;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+
+import java.awt.*;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class ChatController implements Initializable {
+
+    @FXML
+    public TextArea chatArea;
+    @FXML
+    public ListView onlineUsers;
+    @FXML
+    public Button btnSendMessage;
+    @FXML
+    public TextField input;
+
+    @Override
+    public void initialize(URL url, ResourceBundle resourceBundle) {
+        onlineUsers.setItems(FXCollections.observableArrayList("Вася", "Петя"));
+    }
+    public void exit(ActionEvent actionEvent) {
+        Platform.exit();
+    }
+
+    public void showHelp(ActionEvent actionEvent) throws URISyntaxException, IOException {
+        Desktop desktop = Desktop.getDesktop();
+        desktop.browse(new URI("https://docs.google.com/document/d/1wr0YEtIc5yZtKFu-KITqYnBtp8KC28v2FEYUANL0YAM/edit#"));
+    }
+
+
+
+
+    public void Item(ActionEvent actionEvent) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.